applications/luci-app-unbound/luasrc/model/cbi/unbound.lua

```diff
@@ -125,6 +131,15 @@ wfq:value("4", translate("Interface FQDN, All Addresses"))
 wfq:depends({ dhcp_link = "none" })
 wfq:depends({ dhcp_link = "odhcpd" })
 
+exa = s1:taboption("advanced", ListValue, "add_extra_dns", translate("Extra DNS:"),
```

Comment:
LuCI has its own repo and each PR can only address one: openwrt/packages and openwrt/luci. luci-app-unbound will demand unbound to be installed. `LUCI_DEPENDS:=+unbound`